platform/core/src/utils/getImageId.js

```diff
@@ -38,18 +38,14 @@ export default function getImageId(instance, frame, thumbnail = false) {
 
   const renderingAttr = thumbnail ? 'thumbnailRendering' : 'imageRendering';
 
-  if (
-    !instance[renderingAttr] ||
-    instance[renderingAttr] === 'wadouri' ||
-    !instance.wadorsuri
-  ) {
-    let imageId = 'dicomweb:' + instance.wadouri;
+  let imageId = getWADORSImageId(instance, frame, thumbnail); // WADO-RS Retrieve Frame
```

Comment:
I'm not sure I understand the purpose of this change. From what I can tell, this would mean that even if the user has specified that we should be using WADO-URI, if the server and instances support WADO-RS this would use that instead. That doesn't seem like the behaviour we want. What is this trying to fix? I think your other PR where you just fixed the default value for rendering to wadors was sufficient, wasn't it?

Reply:
I was trying to use WADO-RS always by default and just using the WADO-URI if it was not possible to get the imageId from WADO-RS way. The previous else condition was getting WADO-RS by default already, that's why I changed to get everytime. But I agree with you, we can use the default value true for rendering that I fixed in the other PR.